### PR TITLE
handle #8 with try-catch and loop.

### LINKED ISSUE #8: Main menu input crashes on non-numeric text
Here

https://github.com/2002-feb24-net/anthony-project0/blob/master/BalloonParty/BalloonParty.ConsoleApp/Logic/Login.cs#L12

### PR DESCRIPTION
**Pull request by escalonn**
fixes #8

### CODE REVIEW

File: BalloonParty/BalloonParty.ConsoleApp/Logic/Login.cs [hunk 1]
```diff
@@ -10,10 +10,10 @@ public static void mainLogin()
...
             System.Console.WriteLine("Please select if you are a customer, a store, or an admin or press zero to exit.\n1: Customer\n2: Store\n3: Admin\n\n0: Exit");
 
-            if (!int.TryParse(Console.ReadLine(), out int input))
+            int input;
+            while (!int.TryParse(Console.ReadLine(), out input))
             {
-                Console.WriteLine("Invalid input, quitting.");
-                return;
+                Console.WriteLine("Invalid input. Try again");
             }
```
Comment (escalonn) [inline BalloonParty/BalloonParty.ConsoleApp/Logic/Login.cs:17]: can we comment this block of code

File: BalloonParty/BalloonParty.ConsoleApp/Logic/Login.cs [hunk 2]
```diff
@@ -10,10 +10,10 @@ public static void mainLogin()
         {
             System.Console.WriteLine("Please select if you are a customer, a store, or an admin or press zero to exit.\n1: Customer\n2: Store\n3: Admin\n\n0: Exit");
 
-            if (!int.TryParse(Console.ReadLine(), out int input))
+            int input;
+            while (!int.TryParse(Console.ReadLine(), out input))
             {
-                Console.WriteLine("Invalid input, quitting.");
-                return;
+                Console.WriteLine("Invalid input. Try again");
```
Comment (escalonn) [inline BalloonParty/BalloonParty.ConsoleApp/Logic/Login.cs:16]: consider being more descriptive about what the user did wrong.

**Review by escalonn — COMMENTED** [review]
looks great, excellent code

File: BalloonParty/BalloonParty.ConsoleApp/Logic/Login.cs (hunk 1 above)
Comment (GhostParticle) [reply BalloonParty/BalloonParty.ConsoleApp/Logic/Login.cs:17]: we could possibly

**Review by GhostParticle — APPROVED** [review]

**Pull request merged**